web/src/views/events/EventView.tsx

```diff
@@ -441,10 +452,16 @@ function DetectionReview({
           />
         )}
 
-        {itemsToReview == 0 && (
-          <div className="size-full flex flex-col justify-center items-center">
+        {currentItems == null && (
```

Comment:
could this just be `{!currentItems && ( ...`?

Reply:
!current items covers empty list as well I believe

Reply:
never mind looks like it doesn't, so that works out